src/ui/buttons/styles/button.js

```diff
@@ -125,4 +125,27 @@ export const buttonStyle = `
     .${ CLASS.CONTAINER } .${ CLASS.VAULT_HEADER } {
         margin-top: 10px;
     }
+
+    .${ CLASS.BUTTON }.${ CLASS.COLOR }-${ BUTTON_COLOR.BLACK } {
```

Comment:
We already have these defined in this file: https://github.com/paypal/paypal-checkout-components/blob/6127a331598d5d21787d55afd08ff1323e939f90/src/ui/buttons/styles/color.js#L72